src/lib/util/chatHeadImage.ts

```diff
@@ -42,10 +44,11 @@ const names: Record<keyof typeof chatHeads, string> = {
 	marimbo: 'Marimbo',
 	ketKeh: 'Tzhaar-Ket-Keh',
 	gertrude: 'Gertrude',
-	antiSanta: 'Anti-Santa'
+	antiSanta: 'Anti-Santa',
+	bunny: 'Easter Bunny'
 };
 
-export default async function chatHeadImage({ content, head }: { content: string; head: keyof typeof chatHeads }) {
+export async function newChatHeadImage({ content, head }: { content: string; head: keyof typeof chatHeads }) {
```

Comment:
I probably would have named this like chatHeadImageRaw or rawChatHeadImage, but obviously it doesn't matter :)